test/libsolidity/semanticTests/integer/min_max.sol

```diff
@@ -0,0 +1,418 @@
+contract C {
+	function basic() public pure {
```

Comment:
Can you make all the functions return true? Then the expectations are a bit clearer, I think.